stellar/operators.js

```diff
@@ -31,41 +39,21 @@ async function removeOperator(wallet, _, chain, contract, args, options) {
     await broadcast(operation, wallet, chain, 'remove_operator called', options);
 }
 
-async function addGas(wallet, _, chain, contract, args, options) {
-    const operator = addressToScVal(wallet.publicKey());
-    const [sender, messageId, spender, tokenAddress, tokenAmount] = args;
-
-    validateParameters({
-        isNonEmptyString: { sender, messageId, spender, tokenAddress },
-        isValidNumber: { tokenAmount },
-    });
-
-    const target = addressToScVal(chain.contracts?.axelar_gas_service?.address);
-    const method = nativeToScVal('add_gas', { type: 'symbol' });
-    const params = nativeToScVal([
-        addressToScVal(sender),
-        nativeToScVal(messageId, { type: 'string' }),
-        addressToScVal(spender),
-        tokenToScVal(tokenAddress, tokenAmount),
-    ]);
-
-    const operation = contract.call('execute', operator, target, method, params);
-
-    await broadcast(operation, wallet, chain, 'add_gas called', options);
-}
-
 async function collectFees(wallet, _, chain, contract, args, options) {
     const operator = addressToScVal(wallet.publicKey());
-    const [receiver, tokenAddress, tokenAmount] = args;
+    const [receiver] = args;
+    const gasServiceAddress = chain.contracts?.axelar_gas_service?.address;
+    const gasTokenAddress = options.gasTokenAddress || chain?.tokenAddress;
```

Comment:
```suggestion
    const gasTokenAddress = options.gasTokenAddress || chain.tokenAddress;
```
`chain` config can assume to exist

Reply:
updated for all other places